src/style.css

```diff
@@ -0,0 +1,50 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url("./fonts/macfonts/Avenir/Avenir-Book.ttf");
+}
+
+.search {
+  width: 100%;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  font-style: normal;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir-Book, serif;
+  font-weight: 300;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  border-radius: 3px;
+  outline: none;
+  font-weight: 900;
+}
+
+.search-big {
+  height: 70px;
+  background-position: 25px 24px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-small {
+  margin-top: 20px;
+  height: 42px;
+  background-position: 12px 14px;
+  padding-left: 33px;
+  background-size: 11px 11px;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
We don't have any basic search bar, so why we should use there big and small search class names? We have search-input and search-input - small (--small). Therefore, we set all styles that are supposed to be for the "large" input for all inputs in general, and with the modifier small we change only the necessary values to make our input smaller.